### ISSUE TITLE
Investigate ttrt perf build tracy flags

### ISSUE THREAD

**Issue by tapspatel** [tenstorrent/tt-mlir#681]
Reported by Kyle. With this docker image

--docker-image ghcr.io/tenstorrent/tt-mlir/tt-mlir-ird-ubuntu-22-04:latest 

perf mode fails. CI and local testing passes.

**Comment by tapspatel** [comment]
tldr: perf build works on-top of tip on a fresh build and using IRD image

okay so there’s a mixed use of this flag in metal
When building metal, you need to provide ENABLE_TRACY. This will set TRACY_ENABLE in third_party/tt-metal/src/tt-metal/CMakeLists.txt  which builds all the tracy modules. (previously. this was an env flag which got converted into a compile flag, hence the change in my commit at: 298afb510

Perf build succeeds with this IRD docker image --docker-image [ghcr.io/tenstorrent/tt-mlir/tt-mlir-ird-ubuntu-22-04:latest](http://ghcr.io/tenstorrent/tt-mlir/tt-mlir-ird-ubuntu-22-04:latest) .
Commit = mlir latest.

No issues, user had to clean tt-metal-build folder and everything else built as expected.
